src/core/grafana.tf

```diff
@@ -0,0 +1,18 @@
+
+
+resource "azurerm_dashboard_grafana" "example" {
+  name                              = format("%s-grafana", local.project)
```

Comment:
Please use the interpolation format

Reply:
I copied from all the code that is inside the repo; example: https://github.com/pagopa/pagopa-infra/blob/4243840adcf9bc46444cfe27588c6e6d3900000e/src/core/apim.tf#L4.

Is interpolation still ok?

Reply:
yes it is